modules/swagger-codegen/src/main/resources/csharp/api.mustache

```diff
@@ -81,10 +81,10 @@ namespace {{packageName}}.Api
         /// <returns></returns>
         public {{classname}}(String basePath)
         {
-            this.Configuration = new Configuration(new ApiClient(basePath));
+            this.Configuration = new {{packageName}}.Client.Configuration(new ApiClient(basePath));
```

Comment:
I get compilation errors on the SwaggerClient solution in Visual Studio:

![image](https://cloud.githubusercontent.com/assets/109659/15538918/31cb0e7a-224d-11e6-93d2-27beb05ea34c.png)